woocommerce/woocommerce-android
# Change the copy in the upsell card reader banner.

Closes: #7039 


### Description

This PR changes the copy in the upsell card reader banner from 

`Get ready to accept payments with card reader.`

to

`Start selling in person in under 20 minutes with our card reader.`

Discussion here: pdfdoF-ZW-p2#comment-2067

### Testing instructions

1. Ensure the copy makes sense
2. Run the app and ensure the updated copy is reflected in the banner

### Images/gif

<img width="416" alt="Screenshot 2022-07-26 at 10 07 31 AM" src="https://user-images.githubusercontent.com/1331230/180924178-d70d708c-7363-4239-b32c-9e70c458bae8.png">
